samples/InlineInitializationSample/Startup.cs

```diff
@@ -46,7 +48,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             // Write streamlined request completion events, instead of the more verbose ones from the framework.
             // To use the default framework request logging instead, remove this line and set the "Microsoft"
             // level in appsettings.json to "Information".
-            app.UseSerilogRequestLogging();
+            app.UseSerilogRequestLogging(opts =>
+            {
+                opts.GetLogEventLevel = statusCode =>
+                    statusCode == HttpStatusCode.InternalServerError
+                        ? LogEventLevel.Error
+                        : LogEventLevel.Information;
+                // OR opts.GetLogEventLevel = _ => LogEventLevel.Information;
+            });
```

Comment:
I think it would still be nice to show the default, opinionated `UseSerilogRequestLogging()` in the examples, as they'll be used as templates and in most cases the defaults will be appropriate.

Reply:
Sure. no problem.